src/vm/callhelpers.h

```diff
@@ -39,8 +39,13 @@ struct CallDescrData
     // Return value
     //
 #ifdef ENREGISTERED_RETURNTYPE_MAXSIZE
+#ifdef _TARGET_ARM64_
+    // Use NEON128 to ensure proper alignment for vectors.
+    __declspec(align(16)) NEON128 returnValue[ENREGISTERED_RETURNTYPE_MAXSIZE / sizeof(NEON128)];
```

Comment:
Should this use the `DECLSPEC_ALIGN(16)` macro?

Reply:
I was not aware of it.

Reply:
I see that that's what's used in the VM, so I'll change this.